LinearRegression/LinearRegressionWPF/ViewModels/MainWindowViewModel.cs

```diff
@@ -1,291 +1,35 @@
-﻿using System.ComponentModel;
+﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Runtime.CompilerServices;
 using System.Windows.Input;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Linq;
 
-using LinearRegressionWPF.BackendFeatures;
-using LinearRegressionWPF.Commands;
-using LinearRegressionWPF.Models;
 using LinearRegressionBackend.DataProvider;
-using LinearRegressionBackend.MLCommmons;
 using LinearRegressionBackend.MLModel;
+using LinearRegressionBackend.MLCommmons;
+
+using LinearRegressionWPF.BackendDescriptors;
+using LinearRegressionWPF.Models;
+using LinearRegressionWPF.Commands;
+using System.Collections.ObjectModel;
 
 namespace LinearRegressionWPF.ViewModels
 {
-    internal class MainWindowViewModel : INotifyPropertyChanged
+    public class MainWindowViewModel 
     {
+        ObservableCollection<object> _children;
```

Comment:
Why do we need ObservableCollection<object>? Please use simple solution. Please check @09simon solution from the https://github.com/evoRegression/Regression/tree/GaborPlayground/LinearRegression/LinearRegressionWPF branch.,